scripts/build-win.py

```diff
@@ -97,7 +97,7 @@ def gngen(arch, sio_root, ffmpeg_root, ssl_root, msdk_root, quic_root, scheme, t
         gn_args.append('owt_ffmpeg_root="%s"'%(ffmpeg_root))
     if ffmpeg_root or msdk_root or cg_server:
         gn_args.append('rtc_use_h264=true')
-    if msdk_root or cg_server:
+    if msdk_root or cg_server or cg_client:
```

Comment:
we intend to enable this also for ffmpeg_root?

Reply:
We need to align the behavior with H.264. Is there any license issue to enable it?

Reply:
There should be no. We don't compile ffmpeg during our build process, but only provide instruction on how uses integrate their own ffmpeg library.

Reply:
Added `ffmpeg_root`, removed `cg_client`. Same as H.264.